hnji.txt

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Amritsar</title>
+    <link href="mystyle.css" rel="stylesheet"/>
+</head>
+<body>
+<h1> welcome to HTML</h1>
+<h1> welcome to HTML</h1>
+<h1> welcome to HTML</h1>
+<h1> welcome to HTML</h1>
+<h1> welcome to HTML</h1>
+<h1> welcome to HTML</h1>
+<p>
```

Comment:
Please add some tab spaces
Reffer this site
https://www.freeformatter.com/formatters.html